extension/content/components/recipes/RecipeEditor.js

```diff
@@ -131,14 +175,23 @@ export default function RecipeEditor(props) {
             }
           />
         </FormGroup>
+        <FormGroup hidden={!match.params.slug}>
+          <ControlLabel>Import Instuctions</ControlLabel>
+          <Input
+            componentClass="textarea"
+            rows={3}
+            value={importInstructions}
+            onChange={(value) => setImportInstructions(value)}
+          />
+        </FormGroup>
```

Comment:
I'd rather see this field disappear altogether on non-import pages than just be hidden